src/content/tutorials/en/custom-shapes-and-smooth-curves.mdx

```diff
@@ -242,13 +242,7 @@ To create a curved line, we’ll need to use the [`bezierVertex()`](/reference/p
 - Just like we did for the pointed sparkles, we’ll begin our shape with `beginShape()`, and end it with `endShape()`.
 - Any shape with `bezierVertex()` functions must still begin with a `vertex()` function. We’ll call `vertex(x, y)` first. This acts as the first point in our shape.
   - `x` and `y` represent the x-y coordinates for the first vertex in our star.
-- Next, we’ll call the `bezierVertex()` function. Since we’re working on a 2D sketch, the function takes six parameters, ordered as follows: 
-  - the x-coordinate for the first control point
-  - the y-coordinate for the first control point
-  - the x-coordinate for the second control point
-  - the y-coordinate for the second control point
-  - the x-coordinate for the anchor point
-  - the y-coordinate for the anchor point
+- Next, we’ll call the `bezierVertex()` function. In p5.js 2.0, each bezierVertext(x, y) is just one point; you need the first point to anchor, and each curve after that needs 3 points.
```

Comment:
Minor phrasing suggestion:  "each bezierVertext(x, y) is just one point..." --> "call `bezierVertex(x, y) once for each point. You need the first point to anchor, and each curve after that needs 3 points."

(Also the typo fix in "vertext" - "vertex")